src/lib/orionTypes/EntityType.cpp

```diff
@@ -79,10 +79,14 @@ std::string EntityType::render
 {
   std::string  out = "";
 
+  // No metadata filter in this case, an empty vector is used to fulfil method signature.
+  // For attribute filter, we use the ContextAttributeVector itself
+  std::vector<std::string> emptyMdV;
+
   if (typeNameBefore && asJsonOut)
   {
     out += valueTag("name", type, true);
-    out += contextAttributeVector.render(asJsonObject, EntityTypes, true, true, true);
+    out += contextAttributeVector.render(asJsonObject, EntityTypes, contextAttributeVector.vec, emptyMdV, true, true, true);
```

Comment:
That parameter is a `std::vector<ContextAttribute*>` that defines the attributes to be rendered and in which order. In this particular case (due to the lack of filter, I guess), the attributes to be rendered and their order much with the one in the vector itself. That's why `contextAttributeVector.vec` is used. It may seems a bit werid, but it's ok.

NTC